docs/reference/changelog.md

```diff
@@ -0,0 +1,23 @@
+# Changelog
```

Comment:
Please add links to PRs to make it easier to find reason for changes.

Reply:
Should we put at least the main highlights for 1.0.0, compared to 0.9?

Reply:
https://github.com/open-contracting-extensions/public-private-partnerships/pull/210/commits/10f030d32f1a6595ee122cb20488c1b8d9b660a3 adds the PR links

I've added a section on changes in 1.0.0.beta in https://github.com/open-contracting-extensions/public-private-partnerships/pull/210/commits/22abffb4348bc1bb3de2f50d20bbf42b3f0242c3 - IIRC there wasn't a 0.9, we just labelled the initial release as alpha, without a version number.

Reply:
Nevermind – got confused with OC4IDS numbering :)